kibot/fil_field_rename.py

```diff
@@ -6,10 +6,11 @@
 """
 Implements a field renamer
 """
+from __future__ import annotations
```

Comment:
Most of these imports doesn't seem to be needed. This seems to be needed only when using some special cases of the type hints.

Reply:
It's a way to use the new type hinting syntax.  I agree that it isn't used everywhere now, but it allows to add typing hints without looking if the import is already there.

Reply:
But you added it in a compulsive way! Not where is strictly needed.

Reply:
I started adding it by default in other code as well.  So then I can add typing without running into an error later and having to re-edit the file to add this special import. 

I've done several passes of checking code, so this prevents a few ones.